printer_mods/Badnoob/AB-BN/Readme.md

```diff
@@ -64,6 +64,7 @@ This version of the afterburner fan and duct is a drop in replacement to the spe
 \* Cleaner airflow for better overhangs
 
 ### CHANGE Log
+Added support for Zodiac BMO and BMS
```

Comment:
This line and the next line show up as single line on github, either put an empty line between them, or make this into a list

    - Added support for Zodiac BMO and BMS
    - Removed support for Slice.

Reply:
I really need to read all the comments before rerequesting review :)